subctl/issue-182-subctl-a-kubectl-plugin.md

```diff
@@ -0,0 +1,108 @@
+# Support krew way of installing `subctl`
+
+Related Issue:
+[Add subctl as kubectl plugin](https://github.com/submariner-io/enhancements/issues/182)
+
+## Summary
```

Comment:
The summary is copied from various locations such as https://krew.sigs.k8s.io/docs/developer-guide/distributing-with-krew/. Please write a Submariner-focused summary: why is it useful for the Submariner project to make `subctl` available as a `kubectl` plugin installable using `krew`?

Reply:
The idea is to give users background on kubectl plugin and krew.